scripts/lib.sh

```diff
@@ -44,3 +44,32 @@ function start_gateway() {
   $FM_LN1 -k plugin subcommand=start plugin=$FM_BIN_DIR/ln_gateway minimint-cfg=$FM_CFG_DIR &
   sleep 1 # wait for plugin to start
 }
+
+function get_finality_delay() {
+    cat $FM_CFG_DIR/server-0.json | jq -r '.wallet.finality_delay'
+}
+
+function sat_to_btc() {
+    echo "scale=8; $1/100000000" | bc | awk '{printf "%.8f\n", $0}'
```

Comment:
`awk` is needed here because with the method we used before, pegging in > 100mio sat would result in e.g 01.1 for 110mio sat

Reply:
Cannot reproduce:

```
$ echo "scale=8; 110000000/100000000" | bc
1.10000000
$ echo "scale=8; 1100000000/100000000" | bc
11.00000000

Reply:
yes but e.g `echo "scale=8; 100000/100000000" | bc` will output `.001` and not the `0.001` that we need so I did this workaround: `0"$(echo "scale=8; $PEG_IN_AMOUNT/100000000" | bc )")` which always adds a zero in front but if you go >1btc the zero is still there causing an error. `awk` is needed because `bc` does not add the `0` if its < 1

Reply:
the key part here is we dont use `echo "scale=8; 110000000/100000000" | bc` right now but `0"$(echo "scale=8; 110000000/100000000" | bc)"

Reply:
LOL, that is indeed the top-rated stackoverflow answer, what a sad state of affairs :laughing:

Reply:
yeah xd, thats the time for `fish` shell to shine